packages/cli/doc-core/modern.config.ts

```diff
@@ -14,12 +13,29 @@ export default {
   designSystem: tailwindConfig.theme,
   buildConfig: [
     {
+      buildType: 'bundle',
+      format: 'esm',
+      target: 'es2020',
+      outDir: 'dist',
+      sourceMap: true,
+      externals: ['@modern-js/mdx-rs-binding'],
```

Comment:
It seems `@rspack/core` is removed from externals, will it affect the main bundle?